jetcd-examples/jetcd-simple-ctl/pom.xml

```diff
@@ -63,18 +63,19 @@
   <build>
     <plugins>
```

Comment:
add a mainclass to the manifest:
```xml
     <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-jar-plugin</artifactId>
        <version>3.0.2</version>
        <configuration>
          <archive>
            <manifest>
              <mainClass>com.coreos.jetcd.examples.jetcdctl.Main</mainClass>
            </manifest>
          </archive>
        </configuration>
      </plugin>
```

so the examples can use the shorter -jar form `java -jar  target/jetcdctl.jar put foo bar`